src/deploy.ts

```diff
@@ -13,11 +13,11 @@ export async function deploy (args: string[]) {
     const json = JSON.parse(Deno.readTextFileSync(manifestPath))
     const body = JSON.parse(json.body)
     const dirname = path.dirname(manifestPath)
-    toUpload.push(path.join(dirname, body.contract.file))
+    toUpload.push('t|' + path.join(dirname, body.contract.file))
```

Comment:
What is this `t|` and `m|` stuff? Can you add a comment explaining?

Reply:
It's a prefix because we need to know what kind of CID should be used.